src/sidebar/services/frame-sync.js

```diff
@@ -87,6 +89,16 @@ export class FrameSyncService {
      */
     this._guestRPC = new Bridge();
 
+    /**
+     * Channel for sidebar-notebook communication.
+     *
+     * @type {Bridge<SidebarToNotebookEvent,NotebookToSidebarEvent>}
+     */
+    this._notebookRPC = new Bridge();
+    this._notebookRPC.onConnect(() =>
+      this._notebookRPC.call('groupChanged', store.focusedGroup()?.id)
```

Comment:
Related to my investigation about potentially removing `Bridge#onConnect` in future, this call can be moved to just after the `this._notebookRPC.createChannel` call.